src/pages/includes/wallet-info.html

```diff
@@ -0,0 +1,22 @@
+<div class="wallet-info">
+  <!-- TODO -->
+  <div class="actions" *ngIf="(walletNotRegistered || updateStatusError)">
+    <a *ngIf="walletNotRegistered" (click)="recreate()" translate>Tap to recreate</a>
+    <a *ngIf="updateStatusError" (click)="updateAll(true)" translate>Tap to retry</a>
+  </div>
+
+  <!-- TODO -->
+  <ion-spinner class="spinner-dark recent right" icon="crescent" *ngIf="(updatingStatus || updatingTxHistory) && !walletNotRegistered && !updateStatusError && !updateTxHistoryError">
+  </ion-spinner>
+
+  <img style="height:3rem; width:3rem;" *ngIf="wallet.network == 'testnet'" src="assets/img/icon-wallet-testnet.svg">
```

Comment:
The styles should be in the css file, to avoid dirtying the html. The same with the following lines

Reply:
Done